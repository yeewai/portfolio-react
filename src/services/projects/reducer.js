import data from 'projectsData.json';

export const defaultState = data;

const projectsReducer = (state = defaultState, action) => {
    switch(action.type) {
        default:
            return state;
    }
}

export default projectsReducer;
