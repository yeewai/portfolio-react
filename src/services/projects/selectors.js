const sortByDateFn = (a,b) => {
  return new Date(b.date) - new Date(a.date);
}

export const getArt = (state) => state.projects.art.sort(sortByDateFn);
export const getCode = (state) => state.projects.code.sort(sortByDateFn);

export const getByName = ( state, params ) => (
    ["art", "code"].includes(params.projectType)
    ? state.projects[params.projectType].find( p => p.slug === params.projectTitle )
    : null
)
