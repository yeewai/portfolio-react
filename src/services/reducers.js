import { combineReducers } from 'redux';

import { routerReducer } from 'react-router-redux';
import projectsReducer from './projects';

const rootReducer = combineReducers({
    router: routerReducer,
    projects: projectsReducer
});

export default rootReducer;
