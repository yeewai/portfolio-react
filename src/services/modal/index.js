import modalReducer from "./reducer";
import * as modalActions from "./actions";
import * as modalSelectors from "./selectors";


export { modalActions, modalSelectors,};

export default modalReducer;
