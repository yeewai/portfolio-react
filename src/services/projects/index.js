import projectsReducer from "./reducer";
import * as projectsSelectors from "./selectors";

export { projectsSelectors };

export default projectsReducer;
