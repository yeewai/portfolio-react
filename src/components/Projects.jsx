import React from 'react';
import { connect } from 'react-redux';
import { Link } from 'react-router-dom';

import { projectsSelectors } from 'services/projects';

export const Projects = ({projects, projectsType}) => {
    return (
        <article className={`projects ${projectsType} animated fadeIn`}>
            <ul className="allitems ">
                { projects.map( (m, i) =>(
                    <li key={`project-${i}`}>
                        <Link to={`/${projectsType}/${m.slug}`}>
                            <div className="card animated fadeIn">
                                <img className="card-img-top" src={m.thumbnail} alt={m.title} />
                                <div className="card-body">
                                    <h3 className="card-title">{m.title}</h3>
                                    { m.subtitle ? <h4 class="card-subtitle mb-2 text-muted">{m.subtitle}</h4> : null}
                                </div>
                            </div>
                        </Link>
                    </li>
                )) }
            </ul>
        </article>
    );
}

export const mapStateToProps = (state, ownProps) => ({
    projectsType: ownProps.match.params.projectsType,
    projects: ownProps.match.params.projectsType === "art" ? projectsSelectors.getArt(state) : projectsSelectors.getCode(state)
});

export default connect(mapStateToProps)(Projects);
