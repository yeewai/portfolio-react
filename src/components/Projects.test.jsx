import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import sinon from 'sinon';

import { Projects, mapStateToProps } from './Projects';
import { projectsSelectors } from 'services/projects';

Enzyme.configure({ adapter: new Adapter() });

describe("Projects", () => {
    const project1 = {
        images: [
            { url: "a", alt: "b" },
            { url: "c", alt: "d" }
        ],
        title: "Eyyyyyy",
        subtitle: "Sub-eyyyyyyy",
        date: "Some date",
        tools: "Sum tools",
        description: "<p>sup sup sup</p>"
    }

    const project2 = {
        images: [
            { url: "a", alt: "b" },
            { url: "c", alt: "d" }
        ],
        title: "Eyyyyyy2",
        date: "Some date2",
        tools: "Sum tools2",
        description: "<p>sup sup sup</p>"
    }

    it('renders', () => {
        const projects = [project1, project2];

        const wrapper = shallow(<Projects projects={projects} projectsType="/art"/>);
        expect(wrapper).toMatchSnapshot();
    });

    it ('mapStateToProps', () => {
        const sandbox = sinon.sandbox.create();
        sandbox.stub(projectsSelectors, 'getArt').returns("project1");
        sandbox.stub(projectsSelectors, 'getCode').returns("project2");

        expect(mapStateToProps({}, {match: {params: {projectsType: "art"}}}).projects).toEqual("project1");
        expect(mapStateToProps({}, {match: {params: {projectsType: "code"}}}).projects).toEqual("project2");

        sandbox.restore();
    })

})
