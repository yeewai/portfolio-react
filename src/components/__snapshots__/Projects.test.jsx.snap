// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    projects={
        Array [
            Object {
              "date": "Some date",
              "description": "<p>sup sup sup</p>",
              "images": Array [
                Object {
                  "alt": "b",
                  "url": "a",
                },
                Object {
                  "alt": "d",
                  "url": "c",
                },
              ],
              "subtitle": "Sub-eyyyyyyy",
              "title": "Eyyyyyy",
              "tools": "Sum tools",
            },
            Object {
              "date": "Some date2",
              "description": "<p>sup sup sup</p>",
              "images": Array [
                Object {
                  "alt": "b",
                  "url": "a",
                },
                Object {
                  "alt": "d",
                  "url": "c",
                },
              ],
              "title": "Eyyyyyy2",
              "tools": "Sum tools2",
            },
          ]
    }
    projectsType="/art"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <ul
        className="allitems "
>
        <li>
                <Link
                        replace={false}
                        to="//art/undefined"
                >
                        <div
                                className="card animated fadeIn"
                        >
                                <img
                                        alt="Eyyyyyy"
                                        className="card-img-top"
                                        src={undefined}
                                />
                                <div
                                        className="card-body"
                                >
                                        <h3
                                                className="card-title"
                                        >
                                                Eyyyyyy
                                        </h3>
                                        <h4
                                                class="card-subtitle mb-2 text-muted"
                                        >
                                                Sub-eyyyyyyy
                                        </h4>
                                </div>
                        </div>
                </Link>
        </li>
        <li>
                <Link
                        replace={false}
                        to="//art/undefined"
                >
                        <div
                                className="card animated fadeIn"
                        >
                                <img
                                        alt="Eyyyyyy2"
                                        className="card-img-top"
                                        src={undefined}
                                />
                                <div
                                        className="card-body"
                                >
                                        <h3
                                                className="card-title"
                                        >
                                                Eyyyyyy2
                                        </h3>
                                </div>
                        </div>
                </Link>
        </li>
</ul>,
      "className": "projects /art animated fadeIn",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <li>
            <Link
                        replace={false}
                        to="//art/undefined"
            >
                        <div
                                    className="card animated fadeIn"
                        >
                                    <img
                                                alt="Eyyyyyy"
                                                className="card-img-top"
                                                src={undefined}
                                    />
                                    <div
                                                className="card-body"
                                    >
                                                <h3
                                                            className="card-title"
                                                >
                                                            Eyyyyyy
                                                </h3>
                                                <h4
                                                            class="card-subtitle mb-2 text-muted"
                                                >
                                                            Sub-eyyyyyyy
                                                </h4>
                                    </div>
                        </div>
            </Link>
</li>,
          <li>
            <Link
                        replace={false}
                        to="//art/undefined"
            >
                        <div
                                    className="card animated fadeIn"
                        >
                                    <img
                                                alt="Eyyyyyy2"
                                                className="card-img-top"
                                                src={undefined}
                                    />
                                    <div
                                                className="card-body"
                                    >
                                                <h3
                                                            className="card-title"
                                                >
                                                            Eyyyyyy2
                                                </h3>
                                    </div>
                        </div>
            </Link>
</li>,
        ],
        "className": "allitems ",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "project-0",
          "nodeType": "host",
          "props": Object {
            "children": <Link
              replace={false}
              to="//art/undefined"
>
              <div
                            className="card animated fadeIn"
              >
                            <img
                                          alt="Eyyyyyy"
                                          className="card-img-top"
                                          src={undefined}
                            />
                            <div
                                          className="card-body"
                            >
                                          <h3
                                                        className="card-title"
                                          >
                                                        Eyyyyyy
                                          </h3>
                                          <h4
                                                        class="card-subtitle mb-2 text-muted"
                                          >
                                                        Sub-eyyyyyyy
                                          </h4>
                            </div>
              </div>
</Link>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <div
                className="card animated fadeIn"
>
                <img
                                alt="Eyyyyyy"
                                className="card-img-top"
                                src={undefined}
                />
                <div
                                className="card-body"
                >
                                <h3
                                                className="card-title"
                                >
                                                Eyyyyyy
                                </h3>
                                <h4
                                                class="card-subtitle mb-2 text-muted"
                                >
                                                Sub-eyyyyyyy
                                </h4>
                </div>
</div>,
              "replace": false,
              "to": "//art/undefined",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt="Eyyyyyy"
                    className="card-img-top"
                    src={undefined}
/>,
                  <div
                    className="card-body"
>
                    <h3
                                        className="card-title"
                    >
                                        Eyyyyyy
                    </h3>
                    <h4
                                        class="card-subtitle mb-2 text-muted"
                    >
                                        Sub-eyyyyyyy
                    </h4>
</div>,
                ],
                "className": "card animated fadeIn",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Eyyyyyy",
                    "className": "card-img-top",
                    "src": undefined,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <h3
                        className="card-title"
>
                        Eyyyyyy
</h3>,
                      <h4
                        class="card-subtitle mb-2 text-muted"
>
                        Sub-eyyyyyyy
</h4>,
                    ],
                    "className": "card-body",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Eyyyyyy",
                        "className": "card-title",
                      },
                      "ref": null,
                      "rendered": "Eyyyyyy",
                      "type": "h3",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Sub-eyyyyyyy",
                        "class": "card-subtitle mb-2 text-muted",
                      },
                      "ref": null,
                      "rendered": "Sub-eyyyyyyy",
                      "type": "h4",
                    },
                  ],
                  "type": "div",
                },
              ],
              "type": "div",
            },
            "type": [Function],
          },
          "type": "li",
        },
        Object {
          "instance": null,
          "key": "project-1",
          "nodeType": "host",
          "props": Object {
            "children": <Link
              replace={false}
              to="//art/undefined"
>
              <div
                            className="card animated fadeIn"
              >
                            <img
                                          alt="Eyyyyyy2"
                                          className="card-img-top"
                                          src={undefined}
                            />
                            <div
                                          className="card-body"
                            >
                                          <h3
                                                        className="card-title"
                                          >
                                                        Eyyyyyy2
                                          </h3>
                            </div>
              </div>
</Link>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <div
                className="card animated fadeIn"
>
                <img
                                alt="Eyyyyyy2"
                                className="card-img-top"
                                src={undefined}
                />
                <div
                                className="card-body"
                >
                                <h3
                                                className="card-title"
                                >
                                                Eyyyyyy2
                                </h3>
                </div>
</div>,
              "replace": false,
              "to": "//art/undefined",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt="Eyyyyyy2"
                    className="card-img-top"
                    src={undefined}
/>,
                  <div
                    className="card-body"
>
                    <h3
                                        className="card-title"
                    >
                                        Eyyyyyy2
                    </h3>
</div>,
                ],
                "className": "card animated fadeIn",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Eyyyyyy2",
                    "className": "card-img-top",
                    "src": undefined,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <h3
                        className="card-title"
>
                        Eyyyyyy2
</h3>,
                      null,
                    ],
                    "className": "card-body",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Eyyyyyy2",
                        "className": "card-title",
                      },
                      "ref": null,
                      "rendered": "Eyyyyyy2",
                      "type": "h3",
                    },
                    null,
                  ],
                  "type": "div",
                },
              ],
              "type": "div",
            },
            "type": [Function],
          },
          "type": "li",
        },
      ],
      "type": "ul",
    },
    "type": "article",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <ul
          className="allitems "
>
          <li>
                    <Link
                              replace={false}
                              to="//art/undefined"
                    >
                              <div
                                        className="card animated fadeIn"
                              >
                                        <img
                                                  alt="Eyyyyyy"
                                                  className="card-img-top"
                                                  src={undefined}
                                        />
                                        <div
                                                  className="card-body"
                                        >
                                                  <h3
                                                            className="card-title"
                                                  >
                                                            Eyyyyyy
                                                  </h3>
                                                  <h4
                                                            class="card-subtitle mb-2 text-muted"
                                                  >
                                                            Sub-eyyyyyyy
                                                  </h4>
                                        </div>
                              </div>
                    </Link>
          </li>
          <li>
                    <Link
                              replace={false}
                              to="//art/undefined"
                    >
                              <div
                                        className="card animated fadeIn"
                              >
                                        <img
                                                  alt="Eyyyyyy2"
                                                  className="card-img-top"
                                                  src={undefined}
                                        />
                                        <div
                                                  className="card-body"
                                        >
                                                  <h3
                                                            className="card-title"
                                                  >
                                                            Eyyyyyy2
                                                  </h3>
                                        </div>
                              </div>
                    </Link>
          </li>
</ul>,
        "className": "projects /art animated fadeIn",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <li>
              <Link
                            replace={false}
                            to="//art/undefined"
              >
                            <div
                                          className="card animated fadeIn"
                            >
                                          <img
                                                        alt="Eyyyyyy"
                                                        className="card-img-top"
                                                        src={undefined}
                                          />
                                          <div
                                                        className="card-body"
                                          >
                                                        <h3
                                                                      className="card-title"
                                                        >
                                                                      Eyyyyyy
                                                        </h3>
                                                        <h4
                                                                      class="card-subtitle mb-2 text-muted"
                                                        >
                                                                      Sub-eyyyyyyy
                                                        </h4>
                                          </div>
                            </div>
              </Link>
</li>,
            <li>
              <Link
                            replace={false}
                            to="//art/undefined"
              >
                            <div
                                          className="card animated fadeIn"
                            >
                                          <img
                                                        alt="Eyyyyyy2"
                                                        className="card-img-top"
                                                        src={undefined}
                                          />
                                          <div
                                                        className="card-body"
                                          >
                                                        <h3
                                                                      className="card-title"
                                                        >
                                                                      Eyyyyyy2
                                                        </h3>
                                          </div>
                            </div>
              </Link>
</li>,
          ],
          "className": "allitems ",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "project-0",
            "nodeType": "host",
            "props": Object {
              "children": <Link
                replace={false}
                to="//art/undefined"
>
                <div
                                className="card animated fadeIn"
                >
                                <img
                                                alt="Eyyyyyy"
                                                className="card-img-top"
                                                src={undefined}
                                />
                                <div
                                                className="card-body"
                                >
                                                <h3
                                                                className="card-title"
                                                >
                                                                Eyyyyyy
                                                </h3>
                                                <h4
                                                                class="card-subtitle mb-2 text-muted"
                                                >
                                                                Sub-eyyyyyyy
                                                </h4>
                                </div>
                </div>
</Link>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <div
                  className="card animated fadeIn"
>
                  <img
                                    alt="Eyyyyyy"
                                    className="card-img-top"
                                    src={undefined}
                  />
                  <div
                                    className="card-body"
                  >
                                    <h3
                                                      className="card-title"
                                    >
                                                      Eyyyyyy
                                    </h3>
                                    <h4
                                                      class="card-subtitle mb-2 text-muted"
                                    >
                                                      Sub-eyyyyyyy
                                    </h4>
                  </div>
</div>,
                "replace": false,
                "to": "//art/undefined",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <img
                      alt="Eyyyyyy"
                      className="card-img-top"
                      src={undefined}
/>,
                    <div
                      className="card-body"
>
                      <h3
                                            className="card-title"
                      >
                                            Eyyyyyy
                      </h3>
                      <h4
                                            class="card-subtitle mb-2 text-muted"
                      >
                                            Sub-eyyyyyyy
                      </h4>
</div>,
                  ],
                  "className": "card animated fadeIn",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "alt": "Eyyyyyy",
                      "className": "card-img-top",
                      "src": undefined,
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        <h3
                          className="card-title"
>
                          Eyyyyyy
</h3>,
                        <h4
                          class="card-subtitle mb-2 text-muted"
>
                          Sub-eyyyyyyy
</h4>,
                      ],
                      "className": "card-body",
                    },
                    "ref": null,
                    "rendered": Array [
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "Eyyyyyy",
                          "className": "card-title",
                        },
                        "ref": null,
                        "rendered": "Eyyyyyy",
                        "type": "h3",
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "Sub-eyyyyyyy",
                          "class": "card-subtitle mb-2 text-muted",
                        },
                        "ref": null,
                        "rendered": "Sub-eyyyyyyy",
                        "type": "h4",
                      },
                    ],
                    "type": "div",
                  },
                ],
                "type": "div",
              },
              "type": [Function],
            },
            "type": "li",
          },
          Object {
            "instance": null,
            "key": "project-1",
            "nodeType": "host",
            "props": Object {
              "children": <Link
                replace={false}
                to="//art/undefined"
>
                <div
                                className="card animated fadeIn"
                >
                                <img
                                                alt="Eyyyyyy2"
                                                className="card-img-top"
                                                src={undefined}
                                />
                                <div
                                                className="card-body"
                                >
                                                <h3
                                                                className="card-title"
                                                >
                                                                Eyyyyyy2
                                                </h3>
                                </div>
                </div>
</Link>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <div
                  className="card animated fadeIn"
>
                  <img
                                    alt="Eyyyyyy2"
                                    className="card-img-top"
                                    src={undefined}
                  />
                  <div
                                    className="card-body"
                  >
                                    <h3
                                                      className="card-title"
                                    >
                                                      Eyyyyyy2
                                    </h3>
                  </div>
</div>,
                "replace": false,
                "to": "//art/undefined",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <img
                      alt="Eyyyyyy2"
                      className="card-img-top"
                      src={undefined}
/>,
                    <div
                      className="card-body"
>
                      <h3
                                            className="card-title"
                      >
                                            Eyyyyyy2
                      </h3>
</div>,
                  ],
                  "className": "card animated fadeIn",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "alt": "Eyyyyyy2",
                      "className": "card-img-top",
                      "src": undefined,
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        <h3
                          className="card-title"
>
                          Eyyyyyy2
</h3>,
                        null,
                      ],
                      "className": "card-body",
                    },
                    "ref": null,
                    "rendered": Array [
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "Eyyyyyy2",
                          "className": "card-title",
                        },
                        "ref": null,
                        "rendered": "Eyyyyyy2",
                        "type": "h3",
                      },
                      null,
                    ],
                    "type": "div",
                  },
                ],
                "type": "div",
              },
              "type": [Function],
            },
            "type": "li",
          },
        ],
        "type": "ul",
      },
      "type": "article",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
